qemu.go

```diff
@@ -165,6 +165,7 @@ var kernelDefaultParams = []Param{
 	{"iommu", "off"},
 	{"cryptomgr.notests", ""},
 	{"net.ifnames", "0"},
+	{"pci", "lastbus=0"},
```

Comment:
I am wondering if we want that as a default in virtcontainers. You could add this here https://github.com/clearcontainers/runtime/blob/master/Makefile#L83 instead.

Reply:
@devimc - does this option improve boot speed for all machine types?

Reply:
@jodh-intel  for pc just a little, I only can see a big improvement in q35, I guess because of the chipset

Reply:
@devimc @jodh-intel we used to have this option turned on before in 2.1. Somewhere along the way we removed it. 

But then we did not have hotplug support.

So it should be safe to use it once we confirm it does not cause issues for both VFIO, VFIO hotplug and generic network and storage hotplug.